### PR TITLE
Shared: Use orbit-react-native in CloseButton

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: packages/shared/src/buttons/CloseButton.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ type Props = {|
 
 export default function CloseButton({ onPress }: Props) {
   return (
-    <Button onPress={onPress} type="secondary">
```
Comment (RobinCsl) [inline packages/shared/src/buttons/CloseButton.js]: I don't like web APIs in React Native 😄 `onPress` feels more platform agnostic

File: yarn.lock [hunk 2]
```diff
@@ -1518,6 +1518,13 @@
   dependencies:
     ramda "^0.26.1"
 
+"@kiwicom/orbit-react-native@^0.3.2":
+  version "0.3.2"
+  resolved "https://registry.yarnpkg.com/@kiwicom/orbit-react-native/-/orbit-react-native-0.3.2.tgz#4dd99ad7986c04b5ab31bda34ea6a51a871653a9"
+  integrity sha512-VFf+3KsS/FeQ2A71qhIQiEwtyem5g/O3fRRANYZADRrmqXr6x8RVekASZvnYWKHudc+w0OsuhW46d4kFC1dHsA==
+  dependencies:
+    "@kiwicom/orbit-design-tokens" "^0.6.6"
```
Comment (RobinCsl) [inline yarn.lock]: No `styled-components`, halleluja!

File: packages/shared/src/buttons/CloseButton.js (hunk 1 above)
Comment (tbergq) [reply packages/shared/src/buttons/CloseButton.js]: It feels weird to me to add onClick here but, seems to be requirement from orbit that the API is the same as for the web components 🙈

**Comment by tbergq** [comment]
Let me have a second look at it. Definitely the buttons should act the same 🤔

**Comment by tbergq** [comment]
The opacity is matching the buttons in the native app, so I would say it is an improvement, we should just need to use the orbit button everywhere 😄 

The activeOpacity however is coming way to late yes. I want to investigate it a bit.

**Comment by tbergq** [comment]
Ok, definitely not ready yet, there is no ripple effect on android either. I will see if there is some other component that qualifies. 

Closing this MR.

**Pull request closed without merging**